livekit-agents/livekit/agents/llm/realtime.py

```diff
@@ -83,6 +83,8 @@ async def aclose(self) -> None: ...
     "input_speech_stopped",  # serverside VAD
     "input_audio_transcription_completed",
     "generation_created",
+    "session_reconnected",
+    "session_restored",
```

Comment:
I don't we need this event, any reason to keep it? ideally users just use session_reconnected
```suggestion
```